generator/C/include_v0.9/protocol.h

```diff
@@ -34,6 +34,14 @@
 #define MAVLINK_END_UART_SEND(chan, length)
 #endif
 
+/*
+  to get warnings when any WIP message is used, add this:
+  #define MAVLINK_WIP __attribute__((warning("MAVLink work in progress")))
+*/
+#ifndef MAVLINK_WIP
+#define MAVLINK_WIP
+#endif
```

Comment:
Shouldn't the default be to print the warning and a custom define can be used to silence it?

Reply:
Absolutely, but see my question and the response from @tridge above https://github.com/ArduPilot/pymavlink/pull/240#issuecomment-438557184

Do you think we could make the behaviour as we expect but explicitly disable the warning in tests? We don't particularly want to have warnings in these tests - we want to have them in live environments.